SPECS/vim/vim.spec

```diff
@@ -2,14 +2,18 @@
 Summary:        Text editor
 Name:           vim
 Version:        9.0.2121
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        Vim
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Applications/Editors
 URL:            https://www.vim.org
 Source0:        https://github.com/%{name}/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 Patch0:         CVE-2024-22667.patch
+Patch1:         CVE-2024-41957.patch
+Patch2:         fix_save_unnamed_buffer_correctly.patch
```

Comment:
Is there a CVE associated with Patch2?  Is that the correct name?

Reply:
@jslobodzian No this patch is not part of CVE, but a general fix upstream (https://github.com/vim/vim/pull/14398). The patch was added here for the CVE patch to apply cleanly.

Reply:
@Sumynwa can you combine them into one patch file then?

Reply:
@jslobodzian Sure I will.

Reply:
@Sumynwa just waiting on the patch to be combined.

Reply:
@jslobodzian Patches have been merged.